docs/api/_index.md

```diff
@@ -170,6 +172,28 @@ GET /ha-tracker
 
 Displays a web page with the current status of the HA tracker, including the elected replica for each Prometheus HA cluster.
 
+## Debugger
+
+### Pprof
+
+```
+GET /debug/pprof/*
```

Comment:
Could you list all endpoints? The final user may have an hard time figuring out which are supported, even if we linked to pprof doc.

Reply:
Done. Could you check it?